app/views/editor/_editor.html.erb

```diff
@@ -2,7 +2,7 @@
 <%= render :partial => "editor/toolbar", :locals => { :location => :main } %>
 
 <div class="form-group dropzone">
-  <textarea aria-label="Wiki Text" name="body" tabindex="2" class="form-control" id="text-input" rows="20" cols="60"><% if @node && @node.latest && @node.latest.body %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
+  <textarea aria-label="Wiki Text" name="body" tabindex="2" class="form-control" id="text-input-main" rows="20" cols="60"><% if @node && @node.latest && @node.latest.body %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
```

Comment:
Should we change `id` in below line to `class` like you did in above files to keep it consistent?

Reply:
Oh I think it will be okay. This particular instance is in the `editor/editor` partial, which is the legacy editor that is used in `/wiki/new` and `/wiki/edit`... On those pages, there is only one form, so this should be the only `id='text-input-main'` that's on that page.

On pages with multiple forms for each comment (questions, wikis, & research notes), there should still only be one `id='text-input-main'` for the comment form at the bottom of the page.